package.json

```diff
@@ -28,15 +28,18 @@
     "draft-js-import-html": ">=0.4.0",
     "draft-js-import-markdown": ">=0.3.0",
     "draft-js-utils": ">=0.2.0",
-    "immutable": "^3.8.1",
-    "react": "0.14.x || 15.x.x",
-    "react-dom": "0.14.x || 15.x.x"
+    "immutable": "^3.8.1"
+  },
+  "peerDependencies": {
+    "react": "0.14.x || 15.x.x || 16.x.x",
```

Comment:
I like this syntax. Didn't know you could you use `OR`s like this too!

Reply:
Oh, was going to put in a PR to update the React dependency, but looks like it's already here. This'll be needed so we can update Iris to use React 16.